examples/admin-partitions/README.md

```diff
@@ -33,6 +33,7 @@ Clone this repository:
 
 ```console
 $ git clone https://github.com/hashicorp/terraform-aws-consul-ecs.git
+$ git checkout tags/<latest-version>
```

Comment:
I think this should just be, e.g. `git checkout v0.4.0`? Or does `tags/<version>` work for you?

Reply:
they both work for me

Reply:
Oh, this does work for me. I tried it yesterday and must have typoed the tag name or something.